assets/styles/styles.css

```
@@ -0,0 +1,364 @@
+@font-face {
+  font-family: 'GT Walsheim';
+  src: local('GT Walsheim Medium'), local('GTWalsheim-Medium'),
+      url('../font/GTWalsheim-Medium.woff') format('woff');
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'GT Walsheim';
+  src: local('GT Walsheim Regular'), local('GTWalsheim-Regular'),
+      url('../font/GTWalsheim-Regular.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'GT Walsheim';
+  src: local('GT Walsheim Bold'), local('GTWalsheimBold'),
+      url('../font/GTWalsheimBold.woff') format('woff');
+  font-weight: bold;
+  font-style: normal;
+  font-display: swap;
+}
+
+:root {
+  --main-text-clr:#26666B;
+  --main-title-clr:#00343D;
+  --main-bg-clr:#00D8A0;
+  --white: #fff;
+  --gray:#D3DFE1;
+  --green:#0BD8A2;
+  --light-green: #DAFFF5;
+}
+
+h1,
+h2,
+h3,
+p,
+a,
+ul,
+li,
+button {
+  font-family: 'GT Walsheim','Didact Gothic' , sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  font-size: 1.125em;
+  line-height: 144%;
+  color: var(--main-text-clr);
+}
+
+.title-primary {
```

This class name says that there is a `.title` rule in the styles. If it is true, this should be:
```suggestion
.title--primary {
```
If not, it could mean the element only is used in a specific block and should be named having that into account, for instance `.banner__title`

---

it is to be used in multiple elements as a generic rule but there is no single ".title" class. in that case what should be the correct syntax?

---

You can write it as primary-title for instance. If it is something that doesn't depend on anything, the name should give the reader the idea that rule has no dependencies.